components/airtable/new-records.js

```diff
@@ -0,0 +1,59 @@
+const airtable = require('https://github.com/PipedreamHQ/pipedream/components/airtable/airtable.app.js')
+const moment = require('moment')
+const axios = require('axios')
+const _ = require('lodash')
+
+module.exports = {
+  name: "new-records",
+  version: "0.0.1",
+  props: {
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+    airtable,
+    baseId: { type: "$.airtable.baseId", appProp: "airtable" },
+    tableId: { type: "$.airtable.tableId", baseIdProp: "baseId" },
+  },
+  async run(event) {
+      const config = {
+        url: `https://api.airtable.com/v0/${this.baseId}/${encodeURIComponent(this.tableId)}`,
+        headers: {
+          Authorization: `Bearer ${this.airtable.$auth.api_key}`,
+        },
+      }
+
+      let maxTimestamp = ""
```

Comment:
What is up with these empty string defaults?? are there other falsy values you are guarding against?

```
let maxTimestamp
const lastMaxTimestamp = this.db.get("lastMaxTimestamp")
if (lastMaxTimestamp) {
  config.params.filterByFormula = ...
  maxTimestamp = lastMaxTimestamp
}
```

Reply:
done